Description: A patch for output-pstoedit.h before 0.31.1-16
 This patch has been created to represent all changes was made direct
 to the package before implement dpkg-src 3.0(quilt) format, which has 
 a patch system.
 .
 Ensure the use of char for 32/64bits
Author: Edgar Antonio Palma de la Cruz <xbytemx@gmail.com>
Last-Update: 2011-08-07

--- autotrace-0.31.1.orig/output-pstoedit.h
+++ autotrace-0.31.1/output-pstoedit.h
@@ -32,6 +32,6 @@ int     output_pstoedit_invoke_writer
 						at_spline_list_array_type shape,
 						at_msg_func msg_func, 
 						at_address msg_data);
-at_bool output_pstoedit_is_unusable_writer(const at_string name);
+at_bool output_pstoedit_is_unusable_writer(const char *name);
 
 #endif /* Not def: OUTPUTPSTOEDIT_H */
